### DISCUSSION TITLE
How to use EPS clipping path?

### DISCUSSION THREAD

**Discussion by epstrerath** [ImageMagick/ImageMagick#4363] [Help]
hey,

I'm trying to convert an EPS file to JPG with transparent background. The First layer of the EPS is the actual image, the second layer is a low res version that contains the clipping path. 

Is there any way to apply the Clipping path of the second layer to the first high res layer?

My IM version is 7.1.0-10 and I'm trying to convert on centos 8.3.2011.

[example image](https://github.com/ImageMagick/ImageMagick/files/7343758/image.zip)

**Comment by fmw42** [comment]
Can you post an example image?  What is your IM version and platform/OS?

**Reply by epstrerath** [reply]
hey,

thanks for the reply.

My IM version is 7.1.0-10 and I'm trying to convert on centos 8.3.2011.
I attached the example eps image.

[image.zip](https://github.com/ImageMagick/ImageMagick/files/7343758/image.zip)

**Reply by fmw42** [reply]
My Photoshop says there is only one layer. In general, you can extract the clip path from the 8bim profile or copy if from one file to another. But as far a I know Imagemagick has no way to scale that data to a lower resolution. You would have to do that externally.

**Reply by epstrerath** [reply]
what would be the command to use that clip path from the second "scene" on the first?

**Reply by fmw42** [reply]
I am not an expert on this and have never tried it. But this is what I was told to do to copy a clipping path from one image to another

`magick clipping-paths.jpg +write 8BIM:path.bin +delete toes.png -profile path.bin out.png`
